example/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+find_package(OpenGL REQUIRED)
+find_package(Vulkan REQUIRED)
+
+set(LINK_LIBS vkfw glfw Vulkan::Vulkan)
+
+add_executable(boing ${VKFW_EXAMPLE_ROOT}/boing.cpp)
+target_link_libraries(boing PRIVATE ${LINK_LIBS})
+target_include_directories(boing PRIVATE ${VKFW_DEPS_ROOT})
```

Comment:
I think we should take a bit of inspiration from the corresponding glfw targets for these: https://github.com/glfw/glfw/blob/e7ea71be039836da3a98cea55ae5569cb5eb885c/examples/CMakeLists.txt

In particular for MSVC and Apple specific flags.